ee/billing/billing_manager.py

```diff
@@ -355,11 +355,6 @@ def update_org_details(self, organization: Organization, billing_status: Billing
                 org_modified = True
                 sync_org_quota_limits(organization)
 
-        available_features = data.get("available_features", None)
```

Comment:
So we should be good to remove this now because the client and permission checks are all using `available_product_features`?

Reply:
Did one more sweep through the codebase to confirm and ran some manual testing in the UI, including the case you had initially brought up. Everything seems good.